VideoLocker/res/layout/panel_custom_action_bar.xml

```diff
@@ -18,13 +18,13 @@
         android:layout_marginLeft="11dp"
         android:layout_marginStart="11dp"
         android:orientation="vertical"
+        android:contentDescription="@string/cancel_btn"
```

Comment:
I couldn't find "I cancel" or "I cancel button" anywhere in the iOS code. Would you mind pointing me to it. If it is there, it's probably a bug? I'm not sure why it would ever say "I" in the first person?